src/applications/disability-benefits/all-claims/tests/utils/on-submit/purge-toxic-exposure-data.unit.spec.jsx

```diff
@@ -0,0 +1,1122 @@
+import { expect } from 'chai';
+import {
+  EXPOSURE_TYPE_MAPPING,
+  purgeToxicExposureData,
+} from '../../../utils/on-submit';
+
+/**
+ * Unit tests for purgeToxicExposureData function
+ *
+ * Verifies proper removal of orphaned toxic exposure data when users
+ * opt out of sections in the disability compensation form.
+ *
+ * Key behaviors tested:
+ * 1. Feature flag control - only processes when disability526ToxicExposureOptOutDataPurge is true
+ * 2. "None" condition handling - keeps only conditions.none when selected alone
+ * 3. Orphaned data removal - removes details without corresponding selections
+ * 4. Detail retention - preserves details only for locations marked as true
+ * 5. Specify field cleanup - removes otherHerbicideLocations and specifyOtherExposures when invalid
+ * 6. False value preservation - maintains false selections while removing their details
+ * 7. Unknown field preservation - keeps fields not in EXPOSURE_TYPE_MAPPING (not a validation function)
+ */
+describe('purgeToxicExposureData', () => {
```

Comment:
These initial tests are excellent, clear and well organized (reading through them in order builds up understanding of the functionality), nice!